scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py

```diff
@@ -98,8 +107,8 @@ def getDockerPod(docker_id):
 def getPodInfo(podName):
     kubeapi = kubelet_get_api()
 
-    data = Shell.run('kubectl --server=%s get -o json pod %s' % (
-        kubeapi, podName))
+    data = Shell.run('/usr/local/bin/kubectl --server=%s:7080 get -o json pod %s' % (
```

Comment:
I did so because, in one of the env (fedora I think), the path for root
wasn't set to /usr/local/bin.. for the kubelet process. I don't know why
different linux flavors differ in such matters which just adds more pain..

On Mon, Jul 6, 2015 at 10:03 AM, Pedro Marques <notifications@github.com>
wrote:

> In scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py
> <https://github.com/Juniper/contrail-kubernetes/pull/13#discussion_r33956723>
> :
>
> > @@ -98,8 +107,8 @@ def getDockerPod(docker_id):
> >  def getPodInfo(podName):
> >      kubeapi = kubelet_get_api()
> >
> > -    data = Shell.run('kubectl --server=%s get -o json pod %s' % (
> > -        kubeapi, podName))
> > +    data = Shell.run('/usr/local/bin/kubectl --server=%s:7080 get -o json pod %s' % (
>
> We shouldn't hard code /usr/local/bin
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/13/files#r33956723>.
>